3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/validator/impl/HasRestrictedItems.java

```diff
@@ -0,0 +1,19 @@
+package com._6core.platform.shopping.cart.app.validator.impl;
+
+import com._6core.platform.shopping.cart.app.validator.ValidationHandler;
+import com._6core.platform.shopping.cart.domain.ShoppingCartDomain;
+import java.util.List;
+
+/** Example of possible validation rule implementation. Can be changed or superseded. */
+public class HasRestrictedItems extends ValidationHandler<ShoppingCartDomain> {
```

Comment:
Get rid of this class. we dont have validation rule yet. 

you can leave the only one, e.g. NotEmptyValidationRule